modules/rosa-sts-creating-cluster.adoc

```diff
@@ -0,0 +1,93 @@
+[id="rosa-sts-creating-cluster_{context}"]
+= Creating your cluster with STS
+
+You can create an {product-title} cluster with STS using the `rosa` CLI.
+
+.Prerequisites
+
+You have completed the installation prerequisites.
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installs.
+====
+
+.Procedure
+
+. You can create a cluster using the default settings or by specifying custom settings using the interactive mode. To view other options when creating a cluster, enter `rosa create cluster --help`.
++
+Creating a cluster can take up to 40 minutes.
++
+[NOTE]
+====
+Multiple availability zones (AZ) are recommended for production workloads. The default is a single availability zone. Use `--help` for an example of how to set this option manually or use interactive mode to be prompted for this setting.
+====
+
+* To create your cluster with the default cluster settings:
++
+[source, terminal]
+----
+$ rosa create cluster --cluster-name=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+I: Creating cluster with identifier '1de87g7c30g75qechgh7l5b2bha6r04e' and name 'rh-rosa-test-cluster1'
+I: To view list of clusters and their status, run `rosa list clusters`
+I: Cluster 'rh-rosa-test-cluster1' has been created.
+I: Once the cluster is 'Ready' you will need to add an Identity Provider and define the list of cluster administrators. See `rosa create idp --help` and `rosa create user --help` for more information.
+I: To determine when your cluster is Ready, run `rosa describe cluster rh-rosa-test-cluster1`.
+----
+* To create a cluster using interactive prompts:
```

Comment:
s/using interactive prompts / using the interactive prompts

WDYT?

I know, I probably did this in the original ROSA version. Mea culpa :(

Reply:
I agree with the change.